index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main class="maincontainer">
+        <header>
+            <a href="#" class="logo">Crysty</a>
+            <a href="#" class="menu"><img src="images/Union.png" alt="dropdown menu"> </a>
```

Comment:
- [ ] Please make sure the hamburger button has a `<nav>` tag. This is because it's associated with a "dropdown menu", and it will make your code look more professional. 👍

Reply:
- [ ] Kindly make sure to implement the requirement as suggested by the previous reviewer. The hamburger button should be enclosed in the `<nav> tags`.  Example: `<nav>...Your code.....</nav>`